### PR TITLE
Release Erie Changelog

### PR DESCRIPTION
**Pull request by sgolbabaei**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+### ✨ Features
+* Add scale option to markers, to support custom sizing when using the default Marker SVG element.([#9414](https://github.com/mapbox/mapbox-gl-js/pull/9414)) (h/t [adrianababakanian](https://github.com/adrianababakanian))
+
+### 🍏 Improvements
+* Improving the shader compilation speed by manually getting the run-time attributes and uniforms.([#9497](https://github.com/mapbox/mapbox-gl-js/pull/9497))
+* Mapbox gl-js documentation improvements.([#9607](https://github.com/mapbox/mapbox-gl-js/pull/9607))
+
+### 🐞 Bug fixes
+* Fix multi-finger gestures that touch markers.([#9683](https://github.com/mapbox/mapbox-gl-js/pull/9683))
+* Fix files within the style-spec package importing files from the parent repository tree.([#9522](https://github.com/mapbox/mapbox-gl-js/pull/9522))
```
Comment (karimnaaji) [inline CHANGELOG.md:12]: This one can be removed as well, already in 1.10 and 1.9.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
+## 1.11.0-beta.1
+
+### ✨ Features
+* Add scale option to markers, to support custom sizing when using the default Marker SVG element.([#9414](https://github.com/mapbox/mapbox-gl-js/pull/9414)) (h/t [adrianababakanian](https://github.com/adrianababakanian))
+
+### 🍏 Improvements
+* Improving the shader compilation speed by manually getting the run-time attributes and uniforms.([#9497](https://github.com/mapbox/mapbox-gl-js/pull/9497))
+* Mapbox gl-js documentation improvements.([#9607](https://github.com/mapbox/mapbox-gl-js/pull/9607))
```
Comment (karimnaaji) [inline CHANGELOG.md:8]: Same here.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,20 @@
+## 1.11.0-beta.1
+
+### ✨ Features
+* Add scale option to markers, to support custom sizing when using the default Marker SVG element.([#9414](https://github.com/mapbox/mapbox-gl-js/pull/9414)) (h/t [adrianababakanian](https://github.com/adrianababakanian))
```
Comment (asheemmamoowala) [inline CHANGELOG.md:4]:
```suggestion
* Add an option to scale the default `Marker` icon.([#9414](https://github.com/mapbox/mapbox-gl-js/pull/9414)) (h/t [adrianababakanian](https://github.com/adrianababakanian))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,20 @@
...
+### 🍏 Improvements
+* Improving the shader compilation speed by manually getting the run-time attributes and uniforms.([#9497](https://github.com/mapbox/mapbox-gl-js/pull/9497))
+* Mapbox gl-js documentation improvements.([#9607](https://github.com/mapbox/mapbox-gl-js/pull/9607))
+
+### 🐞 Bug fixes
+* Fix multi-finger gestures that touch markers.([#9683](https://github.com/mapbox/mapbox-gl-js/pull/9683))
+* Fix files within the style-spec package importing files from the parent repository tree.([#9522](https://github.com/mapbox/mapbox-gl-js/pull/9522))
+* Fix a bug where using a combined expression of within and other expressions as a filter.([#9611](https://github.com/mapbox/mapbox-gl-js/pull/9611))
+* Fix a bug where map got stuck in a DragRotate interaction if it's mouseup occurred outside of the browser window or iframe.([#9512](https://github.com/mapbox/mapbox-gl-js/pull/9512))
+* Fix potential visual regression on pattern on AMD graphics card vendor.([#9681](https://github.com/mapbox/mapbox-gl-js/pull/9681))
```
Comment (asheemmamoowala) [inline CHANGELOG.md:15]:
```suggestion
* Fix potential visual regression for `*-pattern` properties on AMD graphics card vendor.([#9681](https://github.com/mapbox/mapbox-gl-js/pull/9681))
```

cc @karimnaaji

**Comment by ryanhamley** [comment]
@sgolbabaei ran the changelog draft tool as `changelog -p` to run against the most recent version so it's interesting that so many changes from 1.10.1 got pulled in. Maybe we should add the exact changelog tool command to the release ticket if `changelog -p` isn't correct.

File: CHANGELOG.md (hunk 1 above)
Comment (sgolbabaei) [reply CHANGELOG.md:12]: Couldn't find `9522` in the 1.10 and 1.9.. shouldn't the cl number be in the changelog already?

File: CHANGELOG.md (hunk 2 above)
Comment (sgolbabaei) [reply CHANGELOG.md:8]: can't find other instances of `9607` either..

**Comment by sgolbabaei** [comment]
So I tried `changelog-draft -p` and also `changelog-draft -b release-erie` but both gave me similar result. I still got a lot of older release's changelog items. I removed multiple of them but some still got their way into the PR. Is there a specific command we need to run?

File: CHANGELOG.md (hunk 2 above)
Comment (karimnaaji) [reply CHANGELOG.md:8]: I don't think we mentioned the docs improvement as part of the release changelog, that's probably why it's not in there

File: CHANGELOG.md (hunk 1 above)
Comment (karimnaaji) [reply CHANGELOG.md:12]: This was probably skipped as well from the changelog, but it was published as part of the release.

Comment (karimnaaji) [reply CHANGELOG.md:12]: Actually I just checked this one and it was in the stylespec: https://github.com/mapbox/mapbox-gl-js/blob/master/src/style-spec/CHANGELOG.md#13131

**Pull request merged**